asterisk_mbox_server/__init__.py

```diff
@@ -79,14 +79,16 @@ def _build_cdr(entries, start=0, count=-1, sha=None):
 
     def _send(self, command, msg):
         """Send message prefixed by length."""
+        print('SEND command %s' % (command))
         msglen = len(msg)
         msglen = bytes([msglen >> 24,
                         0xff & (msglen >> 16),
                         0xff & (msglen >> 8),
                         0xff & (msglen >> 0)])
-        self.conn.send(bytes([command]) + msglen + msg)
+        self.conn.send(bytes([command]) + bytes(msglen) + bytes(msg))
```

Comment:
I need to look at this more.  It shouldn't be needed

Reply:
In python 3.6.6 this was a problem (can't concat str to bytes).
In Python 3.7 it is ok.

Reply:
That doesn't seem right.  I have tested the existing code in 3.5 and 3.6.3.  It doesn't seem likely they broke this between 3.6.3 and 3.6.6.  Will look more